src/main/scala/com/metl/model/BackendAdaptor.scala

```diff
@@ -550,3 +551,28 @@ class ResourceCachingAdaptor(sc:ServerConfiguration,cacheConfig:CacheConfig) ext
     super.isReady
   }
 }
+
+class D2LCachingAdaptor(cacheConfig:CacheConfig) {
```

Comment:
Just don't want to prematurely optimise, especially if its at the expense of getting realtime view over changing data.